gcp/api/v1/osv_service_v1.proto

```diff
@@ -97,12 +97,19 @@ message VersionMatchList {
 message VersionMatch {
   // Score in the interval (0.0, 1.0] with 1.0 being a perfect match.
   double score = 1;
+
   // Information about the upstream repository.
   VersionRepositoryInformation repo_info = 2;
   // The OSV identifier.
   osv.Package osv_identifier = 3;
   // CPE 2.3.
   string cpe23 = 5;
+
+  // Definite number of files that have matched
```

Comment:
nit: add periods after these comments here to match the other comments in this file.